src/main/java/com/google/jenkins/plugins/credentials/domains/DomainRequirementProvider.java

```diff
@@ -104,8 +104,8 @@ public static <T extends DomainRequirement> T of(Class<?> type,
 
     try {
       // Add an instance of the annotated requirement
-      return (T) requiresDomain.value().newInstance();
```

Comment:
[deprecated in java9](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Class.html#newInstance())